app/functions_man.py

```diff
@@ -0,0 +1,100 @@
+import time
+from bs4 import BeautifulSoup
+from selenium.webdriver.common.by import By
+from categories_man import driver
+
+
+def find_index(tag, class_, clothesBS):
+    index_text = clothesBS.find(tag, class_).text
+    index_text_list = index_text.split(" ")
+    index = index_text_list[-1]
+    return index
+
+
+def find_price(clothesBS):
+    span = clothesBS.find("div", "product-detail-info__price-amount price")
+    price_with_currency = span.find("span", "price-current__amount").text
+    price_text = price_with_currency.split(" ")[0]
+    if len(price_text) > 6:
```

Comment:
move this 6 to variable (we want to avoid hardcoding)

Reply:
I added num_char_price_text